students/GuyCChan/session04/trigrams.py

```diff
@@ -0,0 +1,82 @@
+import string
+import random
+
+def read_in_data():
+# Open text file and put file body in a list.
+    with open('./sherlock_small.txt', 'rb') as file:
```

Comment:
you don't want to use 'rb' -- that reads it as a binary file -- and you do want text!